source/apiVolontaria/volunteer/models.py

```diff
@@ -209,6 +212,11 @@ def nb_volunteers(self):
     def nb_volunteers_standby(self):
         return self.volunteers.filter(participation__standby=True).count()
 
+    @property
+    def duration(self):
+        duration = self.end_date - self.start_date
+        return Decimal.from_float(duration.total_seconds() / 3600)
```

Comment:
Why not just return `duration` and let the user get what he want in the business logic.

```python
@property
def duration(self):
    """
    Return a datetime.timedelta representing the duraction of the event
    """
    return self.end_date - self.start_date
```

So after that we can just use the datetime.timedelta returned where we want:
```python
event.duration.total_seconds() / 3600
```

This way allow us to do more in the future without having to refactor or create a lot of super specific function.